packages/metadata-service/src/readers/reader.service.ts

```diff
@@ -1,16 +1,21 @@
 import { Injectable } from '@nestjs/common';
 import { DbService } from './db.service';
 import { OzbyService } from './ozby.service';
+import { ChitaiGorodService } from './chitai-gorod.service';
 
 @Injectable()
 export class ReaderService {
-  constructor(private db: DbService, private ozbyReader: OzbyService) {}
+  constructor(
+    private db: DbService,
+    private ozbyReader: OzbyService,
+    private chitaiGorodService: ChitaiGorodService
+  ) {}
 
   async lookup(isbn: string) {
     const existing = await this.db.findMaterial(isbn);
     // Log access action. Don't await
     if (existing !== null) {
-      this.db.logAccess(existing.id);
+      await this.db.logAccess(existing.id);
```

Comment:
It's not really required to be awaited, because we don't use a result of the operation and we don't care about precision here.